Tools/CISupport/ews-build/steps.py

```diff
@@ -53,9 +53,11 @@
 BUG_SERVER_URL = 'https://bugs.webkit.org/'
```

Comment:
Looks like there are several other production host URLs hard coded. Might want to check if they should instead be going to UAT or DEV.